CSharpBasics/Selenium_Advanced/EpamPageWithWaitingWebElementsTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.DevTools.V106.IndexedDB;
+using OpenQA.Selenium.Interactions;
+
+namespace Selenium_Advanced
+{
+    public class EpamPageWithWaitingWebElementsTests : BaseTests
+    {
+        private const string _careersJoinOurTeamPage = "https://www.epam.com/careers/job-listings";
+
+        private const string _careersButtonHeaderMenuMainPageXPath = "//*[@class = 'top-navigation__item-link' and @href = '/careers']";
+        private const string _careersJoinOurTeamLinkDropMenuXPath = "//*[@class = 'top-navigation__main-link' and @href = '/careers/job-listings']";
+        private const string _chooseLanguageButtonMaimPageXPath = "//*[@class = 'location-selector__button']";
```

Comment:
just suggestion: if you create xpath for class name it is better to use keyword contains :)
just due to the fact that classnames changed easily :)